yaksh/templates/yaksh/add_quiz.html

```diff
@@ -37,8 +37,17 @@
 {% if quiz and course_id %}
     {% if quiz.questionpaper_set.get.id %}
     <center>
-    <h4>You can check the quiz by attempting it in the following modes:</h4>
       <a href="{{URL_ROOT}}/exam/manage/designquestionpaper/{{ quiz.id }}/{{quiz.questionpaper_set.get.id}}/{{course_id}}" class="btn btn-primary">View Question Paper</a>
+      {% if user.profile.is_moderator %}
+        {% if quiz.questionpaper_set.get %}
+          <a href="{{URL_ROOT}}/exam/manage/preview_questionpaper/{{quiz.questionpaper_set.get.id}}" class="btn btn-primary" target="_blank">
+            Preview Question Paper
```

Comment:
Also there is "View Question Paper" button on this page, it will good if that name changes now to something like "Edit Question Paper"

Reply:
Done.